### PR TITLE
Adjust acceptance tests for GS

### PR DESCRIPTION
**Pull request by bernerhat**

### CODE REVIEW

File: tests/manage/pv_services/pvc_resize/test_pvc_expansion.py [hunk 1]
```diff
@@ -39,39 +39,46 @@ def setup(self, multi_pvc_factory, pod_factory):
...
-            constants.STATUS_RUNNING,
-        )
-        pods_rbd = helpers.create_pods(
-            self.pvcs_rbd,
-            pod_factory,
-            constants.CEPHBLOCKPOOL,
-            2,
-            constants.STATUS_RUNNING,
-        )
+        if config.COMPONENTS.get("disable_blockpools"):
```
Comment (ebenahar) [inline tests/manage/pv_services/pvc_resize/test_pvc_expansion.py:42]: shouldn't it be the other way around? so if "disable_blockpools" is true we will skip the PVCs and pods creation of RBD?

File: tests/manage/pv_services/pvc_resize/test_pvc_expansion.py [hunk 2]
```diff
@@ -39,39 +39,46 @@ def setup(self, multi_pvc_factory, pod_factory):
...
+                pod_factory,
+                constants.CEPHBLOCKPOOL,
+                2,
+                constants.STATUS_RUNNING,
+            )
+        else:
+            self.pvcs_rbd = []
+            pods_rbd = []
+
+        if config.COMPONENTS.get("disable_cephfs"):
```
Comment (ebenahar) [inline tests/manage/pv_services/pvc_resize/test_pvc_expansion.py:63]: same here for CephFS

File: tests/manage/pv_services/pvc_resize/test_pvc_expansion.py (hunk 1 above)
Comment (bernerhat) [reply tests/manage/pv_services/pvc_resize/test_pvc_expansion.py]: you are totally right i forgot the not in the beginning of the condition, fixed.

**Review by ebenahar — APPROVED** [review]

**Comment by shylesh** [comment]
@bernerhat https://github.com/red-hat-storage/ocs-ci/pull/4627/files I guess this should be useful for your case. Request your feedback.

**Pull request closed without merging**